src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-beta.1
+
+Released 2023-Jun-22
```

Comment:
```suggestion
Released 2023-Jun-23
```

Reply:
God, I feel so stupid... lol

Reply:
It was fine, when you have created PR :)